sections/php.zsh

```diff
@@ -27,7 +27,7 @@ spaceship_php() {
 
   spaceship::exists php || return
 
-  local php_version=$(php -v 2>&1 | grep --color=never -oe "^PHP\s*[0-9.]\+" | awk '{print $2}')
+  local php_version=$(php -r "echo phpversion();" 2>/dev/null)
```

Comment:
This outputs too much information for me in Linux and it's against [our philosophy](https://github.com/denysdovhan/spaceship-prompt/blob/master/CONTRIBUTING.md#philosophy).

![verbose php version](https://user-images.githubusercontent.com/10276208/38362410-6513fecc-38ee-11e8-9afb-5b2a4d19c42a.png)

- How is function [`phpversion()`](https://secure.php.net/manual/en/function.phpversion.php) different from constant [`PHP_VERSION`](https://secure.php.net/manual/en/reserved.constants.php) ?
- How good is the performance improvement ?

Reply:
It would be better if we had only `MAJOR.MINOR.RELEASE` and remove the part including and after `-`. Can we do that without affecting performance ?

Reply:
Here `php` was installed from Ubuntu provided `apt`. Documentation notes `major.minor.release[extra]` as the output for `PHP_VERSION`.